lib/govuk_index/popularity_worker.rb

```diff
@@ -4,41 +4,52 @@ class PopularityWorker < Indexer::BaseWorker
     QUEUE_NAME = "bulk".freeze
     sidekiq_options queue: QUEUE_NAME
 
-    def perform(records, destination_index)
-      actions = Index::ElasticsearchProcessor.new(client: GovukIndex::Client.new(timeout: BULK_INDEX_TIMEOUT, index_name: destination_index))
+    def perform(document_ids, index_name)
+      actions = Index::ElasticsearchProcessor.new(
+        client: GovukIndex::Client.new(timeout: BULK_INDEX_TIMEOUT, index_name:),
+      )
+
+      source_index = IndexFinder.by_name(index_name)
```

Comment:
```suggestion
      index = IndexFinder.by_name(index_name)
```

As we no longer have a concept of source and destination